### PR TITLE
BETA: Register ashishagarwal.is-a.dev

### PR DESCRIPTION
**Pull request by devashish2024**
Added `ashishagarwal.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

**Review by Omxpro — APPROVED** [review]

**Comment by ashishagarwal2023** [comment]
Can it be done?

File: domains/ashishagarwal.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "e-coders",
+        "email": "aasheeshsaumya@gmail.com"
+    },
+    "record": {
+        "CNAME": "e-coders.github.io"
```
Comment (CuteDog5695) [inline domains/ashishagarwal.json:7]: site gives 404 error

**Review by iostpa — CHANGES_REQUESTED** [review]

File: domains/ashishagarwal.json (hunk 1 above)
Comment (ashishagarwal2023) [reply domains/ashishagarwal.json:7]: Oh...damn I forgot it I will fix it by tommorow

**Comment by ashishagarwal2023** [comment]
Wait one sec, I reanamed my github account after this PR. e-coders was my prev name, could it be fixed I gues? Yeah I'll be working on site thought

**Comment by CuteDog5695** [comment]
it can be fixed by editing the .json file from your fork

**Comment by ashishagarwal2023** [comment]
Ok sure

**Comment by ashishagarwal2023** [comment]
I added my new email also

**Review by Omxpro — APPROVED** [review]

**Comment by ashishagarwal2023** [comment]
Hey, I'm done with my site at ashishagarwal2023.github.io. Can it be changed to ashishagarwal.is-a.dev? Thank you also! @CuteDog5695

**Review by iostpa — APPROVED** [review]
seems good to me now

**Comment by ashishagarwal2023** [comment]
image
Huh?

**Comment by ashishagarwal2023** [comment]
Oh I didn't read, ok..

**Pull request merged**